tensorflow/python/keras/backend.py

```diff
@@ -1212,24 +1212,30 @@ def zeros(shape, dtype=None, name=None):
   """Instantiates an all-zeros variable and returns it.
 
   Arguments:
-      shape: Tuple of integers, shape of returned Keras variable
-      dtype: String, data type of returned Keras variable
-      name: String, name of returned Keras variable
+      shape: Tuple or list of integers, shape of returned Keras variable
+      dtype: data type of returned Keras variable
+      name : name of returned Keras variable
 
   Returns:
       A variable (including Keras metadata), filled with `0.0`.
       Note that if `shape` was symbolic, we cannot return a variable,
       and will return a dynamically-shaped tensor instead.
 
-  Example:
+  For Example:
+  
   ```python
-      >>> from keras import backend as K
-      >>> kvar = K.zeros((3,4))
-      >>> K.eval(kvar)
-      array([[ 0.,  0.,  0.,  0.],
-             [ 0.,  0.,  0.,  0.],
-             [ 0.,  0.,  0.,  0.]], dtype=float32)
+  from tensorflow.keras import backend as K
+  kvar = K.zeros((3,4))
+  K.eval(kvar)
+  array([[ 0.,  0.,  0.,  0.],
```

Comment:
There is no longer a separation between inputs and outputs here

Reply:
I didn't quite understand your comment here. But I modified it little bit. Please check and let me know whether my changes are fine or not. Thanks!